Sources/ApolloCodegenLib/Frontend/JavaScript/src/index.ts

```diff
@@ -46,8 +48,8 @@ export function loadSchemaFromIntrospectionResult(
 }
 
 export function loadSchemaFromSDL(source: Source): GraphQLSchema {
-  const document = parse(source);
-
+  let document = parse(source);
```

Comment:
I think this is a remnant from the earlier change before we discovered `extendSchema`. It doesn't look like `document` is modified anywhere so it can remain `const` defined?

Reply:
Good catch